docs/en/guides/using-with-vuex.md

```diff
@@ -240,23 +240,27 @@ describe('Modules.vue', () => {
     }
 
     store = new Vuex.Store({
-      state,
-      actions,
-      getters: module.getters
+      modules: {
+        mymodule: {
+          state,
+          actions,
+          getters: module.getters
+        }
+      }
     })
   })
 
   it('calls store action "moduleActionClick" when button is clicked', () => {
-    const wrapper = shallowMount(Modules, { store, localVue })
+    const wrapper = shallowMount(MyComponent, { store, localVue })
     const button = wrapper.find('button')
     button.trigger('click')
     expect(actions.moduleActionClick).toHaveBeenCalled()
   })
 
   it('Renders "state.inputValue" in first p tag', () => {
-    const wrapper = shallowMount(Modules, { store, localVue })
+    const wrapper = shallowMount(MyComponent, { store, localVue })
     const p = wrapper.find('p')
-    expect(p.text()).toBe(state.module.clicks.toString())
+    expect(p.text()).toBe(state.clicks.toString())
```

Comment:
@eddyerburgh I guess that's wrong right?

Reply:
Yes, can you change it in the example repo too please? https://github.com/eddyerburgh/vue-test-utils-vuex-example/blob/master/test/unit/specs/MyComponent.spec.js#L47

Reply:
wait, so my change is correct? not sure

Reply:
Yes your change is correct

Reply:
https://github.com/eddyerburgh/vue-test-utils-vuex-example/pull/10